Library/Homebrew/brew.sh

```diff
@@ -12,7 +12,19 @@ then
   then
     export LC_ALL="en_US.UTF-8"
   else
-    export LC_ALL="C.UTF-8"
+    utf8_locales=$(locale -a | grep --extended-regexp '\.(utf|UTF-)8')
```

Comment:
Would be good to store `locale -a` in a variable so it doesn't need re-called below. Also: could this use a Bash regex instead?